app/src/androidTest/java/com/duckduckgo/app/notification/AndroidNotificationSchedulerTest.kt

```diff
@@ -98,17 +97,29 @@ class AndroidNotificationSchedulerTest {
         assertNoUnusedAppNotificationScheduled()
     }
 
+    @Test
+    fun allDeprecatedWorkIsCancelledUponStart() = runBlocking<Unit> {
```

Comment:
Our tests names follow a `when<condition>Then<expectedResult>` naming convention. This test doesn't seem to validate what the name suggest e.g if I delete the new call to `cancelAllUnnecessaryWork` this test still passes.

Reply:
good catch, I will modify the test to enqueue a few deprecated jobs!

Reply:
and change the name of course